src/dmd/access.d

```diff
@@ -60,7 +60,7 @@ bool checkAccess(AggregateDeclaration ad, Loc loc, Scope* sc, Dsymbol smember)
                 return false;
         }
 
-        ad.error(loc, "member `%s` is not accessible%s", smember.toChars(), (sc.flags & SCOPE.onlysafeaccess) ? " from `@safe` code".ptr : "".ptr);
+        ad.error(loc, "`%s %s` is not accessible%s", smember.kind(), smember.toChars, (sc.flags & SCOPE.onlysafeaccess) ? " from `@safe` code".ptr : "".ptr);
```

Comment:
I think the first format specifier should be outside the opening backtick. i.e. the format string to the error call should be 
```
"%s `%s` is not accessible%s"
```
not 
```
"`%s %s` is not accessible%s"
```

Reply:
cc @RazvanN7

Reply:
Yes, I agree.